api/types/client.go

```diff
@@ -1,17 +1,31 @@
 package types
 
 type VanConnectorCreateOptions struct {
-	Name string
-	Cost int32
+	SkupperNamespace string
```

Comment:
I was sure I had already made this comment, but can't see it anywhere. Perhaps I forgot to click the submit or something and it got lost.

My instinct is that it would be nicer to have the namespace as another parameter to each of the client calls (i.e. along side the Context. That is more uniform and to me the namespace is a distinct option. If left as "" it would then imply that the cli Namespace would be used. Even nicer might by to have two variants of each client method, e.g. van_router_create() and van_router_create_ns(), where the second option would have the namespace argument.

That said, this comment is really more of a comment on the API not the functionality. I would be fine leaving it as something to discuss later.

Reply:
Definitely agree with your observations. It would be useful to discuss API conventions generally (e.g. such as multiple variants) and then work towards revising (maybe right after 0.3).